cmd/operator-sdk/test/local.go

```diff
@@ -82,6 +83,9 @@ func newTestLocalCmd() *cobra.Command {
 		"Use a different operator image from the one specified in the namespaced manifest")
 	testCmd.Flags().StringVar(&tlConfig.localOperatorFlags, "local-operator-flags", "",
 		"The flags that the operator needs (while using --up-local). Example: \"--flag1 value1 --flag2=value2\"")
+	testCmd.Flags().BoolVar(&tlConfig.skipCleanupOnError, "skip-cleanup-error", false,
+		"The clean up function responsible to remove all artefacts will be skipped "+
+			"if an error be found and it be set as true.")
```

Comment:
maybe:  "Cleanup of all artifacts will be skipped if an error is found and set to true." ?

Reply:
But the skip just occurs in the failures scenarios. Could you help us to re-phrase it better?

Reply:
"If set as true, the cleanup function responsible to remove all artefacts will be skipped if an error is faced."

Reply:
Shows solved.